scheduler.py

```diff
@@ -89,6 +91,85 @@ def string_schedule(self):
 				date = date + timedelta(days=1)
 		return string
 
+	def json_schedule(self):
+		request_json = []
+
+		# Loop through schedule to create the skeleton JSON dictionaries
+
+		# Date for the first loop
+		json_date = self.start_date
+		if json_date is not None:
+			json_date = json_date + timedelta(days=1)
```

Comment:
why +1 days? wouldn't that push the start date to 1 day later than intended?

Reply:
idk. I stole it from your logic